ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -176,3 +178,48 @@ def test_tensorflow_slogdet(
         fn_tree="linalg.slogdet",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# tensordot
+@handle_cmd_line_args
+@given(
+    dtype_x1_x2_axis=_get_dtype_value1_value2_axis_for_tensordot(
+        available_dtypes=helpers.get_dtypes("numeric"),
+    ),
+    as_variable=st.booleans(),
```

Comment:
`as_variable` and `native_array` are handled by `@handle_cmd_line_args` and can be removed from `@given`.